src/commands/functions-env-clone.ts

```diff
@@ -0,0 +1,46 @@
+import * as clc from "cli-color";
+
+import { Command } from "../command";
+import { requirePermissions } from "../requirePermissions";
+import * as fenv from "../functions/env";
+import * as getProjectId from "../getProjectId";
+import * as utils from "../utils";
+
+export default new Command("functions:env:clone")
+  .description("clone environment variables from another project")
+  .option("--from <projectId>", "the project from which to clone environment variables")
+  .option("--only <keys>", "a comma-separated list of keys to clone")
+  .option("--except <keys>", "a comma-separated list of keys to not clone")
+  .before(requirePermissions, [
+    "firebase.envstores.create",
+    "firebase.envstores.get",
+    "firebase.envstores.list",
+    "firebase.envstores.update",
+  ])
+  .action(async (_: string[], options: any) => {
```

Comment:
Why do we need this? I checked a few other commands that don't take args, and none of them do this.

Reply:
Great to know. I didn't know not having args was an option.